port/win/io_win.cc

```diff
@@ -624,10 +626,8 @@ IOStatus WinSequentialFile::PositionedRead(uint64_t offset, size_t n,
     return IOStatus::NotSupported("This function is only used for direct_io");
   }
 
-  if (!IsSectorAligned(static_cast<size_t>(offset)) || !IsSectorAligned(n)) {
-    return IOStatus::InvalidArgument(
-        "WinSequentialFile::PositionedRead: offset is not properly aligned");
-  }
+  assert(IsSectorAligned(GetSectorSize(), static_cast<size_t>(offset)));
+  assert(IsSectorAligned(GetSectorSize(), static_cast<size_t>(n)));
```

Comment:
Should IsSectorAligned be a member function of WinFileData?

Reply:
yes, that would be better.